plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_alerts/task.py

```diff
@@ -59,7 +60,7 @@ def run(self, params={}, state={}, custom_config: dict = {}):  # pylint: disable
 
         try:
             alert_limit = self.get_alert_limit(custom_config=custom_config)
-            now_time = self._get_current_time()
+            now_time = datetime.utcnow()
```

Comment:
Wonder if that's not deprecated in `datetime`

`from datetime import UTC`

```suggestion
            now_time = datetime.now(UTC)
```